docker/main/rootfs/usr/local/nginx/conf/nginx.conf

```
@@ -299,6 +300,18 @@ http {
                 add_header Cache-Control "public";
             }
 
+            location ~ ^/.*-([A-Za-z0-9]+)\.webmanifest$ {
+                access_log off;
+                expires off;
+                add_header Cache-Control "public, must-revalidate" always;
```

I was wrong. The webmanifest is versioned with a hash. This should be the same as other assets.
```suggestion
                expires 1y;
                add_header Cache-Control "public";
```